### ISSUE TITLE
How do I launch windows application with runas different user option using WinAppDriver

### ISSUE THREAD

**Issue by sksameer18** [microsoft/WinAppDriver#1791]
Is is possible to launch windows application runas different user using WinAppDriver?

My desktop application always we launch with run as different user option, and I am looking for options how can we automate in this situations?

Or Is is possible to attach below process code to WinAppDriver, In below case we can launch our application but how this WinAppDriver session maintains to this to find elements?

System.Diagnostics.Process proc = new System.Diagnostics.Process();
System.Security.SecureString ssPwd = new System.Security.SecureString();
proc.StartInfo.UseShellExecute = false;
proc.StartInfo.FileName = ApplicationPath + "\\" + ApplicationName;
proc.StartInfo.WorkingDirectory = ApplicationPath;
proc.StartInfo.Arguments = Arguments;
proc.StartInfo.Domain = Domain;
proc.StartInfo.UserName = Username;
proc.StartInfo.Password = Password;
proc.Start();

**Comment by anunay1** [comment]
Yes you can start the application with the code and acquire a session with the opened window? Please check #1600
